react-native-libraries.json

```
@@ -12047,5 +12047,17 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/dream-sports-labs/react-native-fast-image",
+    "examples": [
```

Hello @shubh73, thank you for adding new entry to the directory! 👍 

Sorry for the unexpected CI failure, the initial commit you have made was correct. After adding back `npmPkg` the CI check should pass:

```suggestion
    "npmPkg": "@d11/react-native-fast-image",
    "examples": [
```

---

Hey, thanks for the prompt response and for sorting the CI hiccup :)